pkgs/development/python-modules/tensorflow/default.nix

```
@@ -30,7 +32,27 @@
 }:
 
 let
-  inherit (cudaPackages) cudatoolkit cudaFlags cudnn nccl;
+  originalStdenv = stdenv;
+in
+let
+  # Tensorflow looks at many toolchain-related variables which may diverge.
```

```suggestion
  originalStdenv = stdenv;

  # Tensorflow looks at many toolchain-related variables which may diverge.
```

`let ... in let ... in` is redundant

---

Strangely, it doesn't, and I find it unintuitive xD

---

Lol. I don't get infinite recursion locally, but ofborg does https://gist.github.com/GrahamcOfBorg/ebca9d8eefb3ed53e1f8ea70ee211cf9

---

ahhh I see what you mean. hmm maybe we shouldn't reassign `stdenv` then, eg call it `stdenv'` or something?

---

Maybe. The derivation was already using `stdenv` in several places, and although I modify most of  them any way, I didn't feel like changing the name. For one thing, someone might not notice that there's `stdenv'` and introduce the old `stdenv` where it doesn't belong. Idk

---

Fair enough, either way seems totally acceptable imho